packages/story-editor/src/app/story/utils/getStoryPropsToSave.js

```diff
@@ -49,19 +50,7 @@ function getStoryPropsToSave({ story, pages, metadata, flags }) {
     pages,
     ...propsFromStory,
     ...terms,
-    featuredMedia,
-    meta: {
-      web_stories_publisher_logo: story.publisherLogo?.id,
-      web_stories_products: products,
-      web_stories_poster: featuredMedia.isExternal
-        ? {
-            url: featuredMedia.url,
-            width: featuredMedia.width,
-            height: featuredMedia.height,
-            needsProxy: featuredMedia.needsProxy,
-          }
-        : undefined,
-    },
+    products,
```

Comment:
Does this influence the API, too?

Reply:
No, because `getStorySaveData`, which is called with the results of this function, still returns the data in the same format (with `meta: { .... }`)